docs/01_getting_ready_for_class.md

```diff
@@ -39,7 +39,9 @@ $ git --version
 git version 2.11.0
 ```
 
-Anything over 2.9.2 will work for this class!
+In general, we recommend you stay within two minor versions of the [latest version](https://git-scm.com/downloads) of Git. The minor version is the second number in the series `x.x.x`.
+
+We suggest having version [2.29.x](https://raw.githubusercontent.com/git/git/master/Documentation/RelNotes/2.29.1.txt) or more recent if you're using Git for Windows or Git Bash.
```

Comment:
Should we be more specific here like we are on the [Git LFS page](tps://github.com/githubtraining/training-manual/blame/master/docs/26_git_lfs.md#L60)? It’s my understanding that anything less than 
`2.29.2.windows.2` contains a version of Git LFS with a known critical vulnerability: https://github.com/git-lfs/git-lfs/security/advisories/GHSA-4g4p-42wc-9f3m.

```suggestion
We suggest having version [2.29.2](https://github.com/git-for-windows/git/releases/tag/v2.29.2.windows.2) or more recent if you're using Git for Windows or Git Bash.
```